regression-test/data/query_p0/sql_functions/datetime_functions/test_func_time.out

```diff
@@ -0,0 +1,70 @@
+-- This file is automatically generated. You should know what you did if you want to edit this
+-- !sql_const --
+12:12:12
+
+-- !sql_const_scale1 --
+12:12:12
+
+-- !sql_const_scale2 --
+12:12:12
+
+-- !sql_const_scale3 --
+12:12:12
+
+-- !sql_const_scale4 --
+12:12:12
+
+-- !sql_const_scale5 --
+12:12:12
+
+-- !sql_const_scale6 --
+12:12:12
```

Comment:
why the result doesn't have scale equal to input datetime? maybe something wrong in `ComputeSignatureHelper`? could you figure it out?